kubeadm/scripts/Install-Containerd.ps1

```diff
@@ -77,21 +77,19 @@ $env:Path += ";$global:ConainterDPath"
 containerd.exe config default | Out-File "$global:ConainterDPath\config.toml" -Encoding ascii
 #config file fixups
 $config = Get-Content "$global:ConainterDPath\config.toml"
-$config = $config -replace "bin_dir = (.)*$", "bin_dir = `"c:/opt/cni/bin`""
-$config = $config -replace "conf_dir = (.)*$", "conf_dir = `"c:/etc/cni/net.d`""
+$config = $config -replace "bin_dir = (.)*$", "bin_dir = `"C:\\Program Files\\containerd\\cni\\bin`""
+$config = $config -replace "conf_dir = (.)*$", "conf_dir = `"C:\\Program Files\\containerd\\cni\\conf`""
```

Comment:
This doesn't align with the configuration for calico: https://github.com/kubernetes-sigs/sig-windows-tools/blob/6d113824ba7bda0b5c449883c4fc3ac97fa9ea56/hostprocess/calico/calico.yml#L164-L169

This will result in the wrong CNI configuration being used.  We need to align these two values.

Reply:
Forgot to update the calico.yml as well

Reply:
It is better to leave that as it is. Apparently, changing the containerd CNI location will result in a lot more problems to solve than just leaving it as it is. CNIs are configured to have that location as default and changing it will result in even more changes, for the moment it is better to leave it as it is.

Reply:
Makes sense to me to leave it as is.